src/script/miniscript.cpp

```diff
@@ -11,6 +11,16 @@
 #include <assert.h>
 
 namespace miniscript {
+
+bool IsTapscript(MiniscriptContext ms_ctx)
+{
+    switch (ms_ctx) {
+        case MiniscriptContext::P2WSH: return false;
```

Comment:
Is there a reason why the only two options are P2WSH and P2TR? Should there be a case statement for P2SH (`sh`) too?

Reply:
Because the current codebase only support P2WSH, there is no support for legacy P2SH.